scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -254,43 +254,62 @@ it is useful e.g. when having multiple lanes with the same name, but with differ
 }
 
 export class LaneChangeScopeCmd implements Command {
-  name = 'change-scope <remote-scope-name>';
+  name = 'change-scope <lane-name-or-remote-scope> <remote-scope-name>';
   description = `changes the remote scope of a lane`;
-  extendedDescription = 'NOTE: available only before the lane is exported to the remote';
+  extendedDescription = `If no lane is specified (i.e. only one argument provided) this will update the current lane.
+NOTE: available only before the lane is exported to the remote`;
   alias = '';
-  options = [
-    [
-      'l',
-      'lane-name <lane-name>',
-      'the name of the lane to change its remote scope. if not specified, the current lane is used',
-    ],
-  ] as CommandOptions;
+  options = [] as CommandOptions;
   loader = true;
   migration = true;
+  example: [
+    {
+      cmd: 'lane change-scope my-org.other-scope';
+      description: 'change the scope of the current lane to my-org.other-scope';
+    },
+    {
+      cmd: 'lane change-scope my-specific-lane my-org.other-scope';
+      description: "change the scope of lane 'my-specific-lane' to my-org.other-scope";
+    }
+  ];
 
   constructor(private lanes: LanesMain) {}
 
-  async report([remoteScope]: [string], { laneName }: { laneName?: string }): Promise<string> {
-    const { remoteScopeBefore, localName } = await this.lanes.changeScope(remoteScope, laneName);
+  async report([laneNameOrRemoteScope, remoteScope]: [string, string]): Promise<string> {
+    const remoteScopeName = remoteScope ?? laneNameOrRemoteScope;
+    const laneToChange = remoteScope ? laneNameOrRemoteScope : undefined;
+    const { remoteScopeBefore, localName } = await this.lanes.changeScope(remoteScopeName, laneToChange);
```

Comment:
now here you can pass the laneName and in the changeScope method check: if the laneName === 'currentLane' return the local and if not return the name of the other lane

Reply:
i dont need to change the changeScope method, it will ignore the laneName if the value is undefined

Reply:
and it's more confusing to have a check of laneName === scopeName, the two shouldn't be related...
i'm keeping a tertiary here to calculate lane name and will use the nice default assignment you suggested for the destructuring